python/ray/experimental/state.py

```diff
@@ -345,10 +348,22 @@ def task_profiles(self):
         executions of that task. The second element is a list of profiling
         information for tasks where the events have no task ID.
     """
+    if start == None:
+      start = 0
+    if end == None:
+      end = time.time()
+    if num == None:
+      num = sys.maxsize
     task_info = dict()
     event_names = self.redis_client.keys("event_log*")
```

Comment:
I guess these are no longer event names but rather the names of different event log sets, right? Shall we rename `event_names` -> `event_log_sets` or something like that?